ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py

```diff
@@ -1489,6 +1489,37 @@ def test_jax_numpy_fmin(
     )
 
 
+# fabs
+@handle_frontend_test(
+    fn_tree="jax.numpy.fabs",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
You need to test for positive and negative numbers. Make sure the floats are signed

Reply:
Hello @ZoeCD! As I understand floating point data types are always signed (can hold positive and negative values) and thus it should not be a problem. If I try to use "numeric" type instead it shows error that unsigned integer types are used and therefore I think the "float" option is the best in this case. Can you verify if this is the case?